doc/guides/device_mgmt/dfu.rst

```diff
@@ -50,3 +50,169 @@ in the `MCUboot with Zephyr`_ documentation page on the MCUboot website.
 .. _MCUboot boot loader: https://mcuboot.com/
 .. _MCUboot with Zephyr: https://mcuboot.com/mcuboot/readme-zephyr.html
 .. _MCUboot GitHub Project: https://github.com/runtimeco/mcuboot
+
+
+.. _atmel_sam_ba_bootloader:
```

Comment:
Hey, I'm sorry about this but I didn't notice the first time I reviewed this PR that the documentation was going into this file.

I don't think it's really appropriate to put this here. This is about a west flash runner, not device management.

Can I suggest putting this in guides/debugging/host-tools.rst instead and generalizing that page to "flashing and debugging", since bossac only supports flashing?

@MaureenHelm -- would that work for you?

Reply:
Yes

Reply:
Done!